content/layout/intrinsic-4_grouped-listview/index.md

```diff
@@ -0,0 +1,404 @@
+---
+page_type: guide
+title: Grouped ListView Layout
+author: adamargyle
+description:
+tags:
+- Layout
+- Design
+- CSS
+web_lighthouse: N/A
+web_updated_on:
+web_published_on:
+wf_blink_components: Blink>Accessibility
+---
+
```

Comment:
As mentioned in Overview comment-- to include a visual of the parts of the page that map to guides, I think it could be helpful to repeat that same image, calling attention to the part that is covered in this guide, specifically the Group ListView layout.

Reply:
tracked! 👍